src/build-instructions.md

```diff
@@ -292,3 +292,39 @@ If you try to use an unsupported file system, this error will be printed:
 ```
 Cannot mount overlay: Invalid argument
 ```
+
+### SSE3 Support
+
+When running on older CPUs without SSE3 Support, you might run into issues like: 
+
+```
+user$ darling shell
+Bootstrapping the container with launchd...
+Cannot open mnt namespace file: No such file or directory
+```
+
+or more likely
+
+```
+user$ sudo darling shell
```

Comment:
I don't like the implication of including a command like `sudo darling shell` in the wiki. I don't want the wiki to imply that this is an okay thing to do.

I recommend removing the `user$ darling shell` and `user$ sudo darling shell` part.